src/ophys_etl/transforms/convert_rois.py

```diff
@@ -72,6 +71,14 @@ class BinarizeAndCreateROIsInputSchema(ArgSchema):
                      "provided will use default function value of 0.1.")
     )
 
+    longest_edge_threshold = Int(
+        default=50,
+        validate=Range(min=0),
+        description=("ROIs with a longest edge (height or width of "
+                     "the bounding box) larger than this will not be "
```

Comment:
Equal to or greater, not `larger`.

Reply:
replaced with a more accurate description